spec/cli/commands/ip_spec.rb

```diff
@@ -4,4 +4,63 @@
 
 describe Ronin::CLI::Commands::Ip do
   include_examples "man_page"
+
+  let(:ipcmd) { Ronin::CLI::Commands::Ip.new}
+  let(:localhost) { "127.0.0.1" }
+
+  describe "-r" do
+    it "will reverse an IPv4 address" do
+      ipcmd.options[:reverse] = true
```

Comment:
I would probably move these setup lines into `before { }` blocks.

Reply:
Attempting to move these to a `before` block by wholesale setting the `options` variable caused issues due to the `if` blocks present in `#format_ip`.  Naturally, it triggered based on the first option that was set.  I personally dont' think it's too messy setting them with each test but I'm definitely open to advice here.  I haven't touched Ruby in four or five years so I'm out of practice.

Reply:
@postmodern are you fine with `options` being set like this given the conditional logic issue I had?  If so, I'll resolve this comment.

Reply:
@cmhobbs it should be fine provided the `before` is grouped with the tests.

```ruby
describe "#format_ip" do
  context "when --reverse is given" do
    before { subject.options[:reverse] = true }
    
    it "bla bla bla"
      expect(subject.format_ip(ip)).to eq(...)
    end
  end
end
```

Reply:
I initially misunderstood your first comment.  Thanks for the example.  I'll make some edits.